### ISSUE TITLE
iOS Voice control cannot delete line

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#77274]
```dart
import 'package:flutter/cupertino.dart';
import 'package:flutter/material.dart';
void main() {
  runApp(
    MaterialApp(
      home: Scaffold(
        appBar: AppBar(
          title: Text('Expansion Tile Splash'),
          centerTitle: true,
        ),
        body: TextField(),
      ),
    ),
  );
}
```

1. in iOS, turn on voice control
2. tap text field to start editing
3. speak "type how are you"
4. speak "delete line"

expect: the entire line is deleted
actual: only the word "you" was deleted

**Comment by chunhtai** [comment]
speaking 
```
delete word
delete paragraph
delete character
delete document
```
seem to work fine.

**Comment by rajrawat37** [comment]
I want to work on this issue

**Comment by chunhtai** [comment]
Hi @rajrawat37 I already have a fix for this issue.
If you are interested in iOS voice access, there is another similar issue https://github.com/flutter/flutter/issues/77614.

**Comment by TahaTesser** [comment]
Hi 

<details> 
<summary>code sample</summary> 

```dart
import 'package:flutter/cupertino.dart';
import 'package:flutter/material.dart';

void main() {
  runApp(
    MaterialApp(
      home: Scaffold(
        appBar: AppBar(
          title: Text('Expansion Tile Splash'),
          centerTitle: true,
        ),
        body: TextField(),
      ),
    ),
  );
}
``` 

</details>


<details> 
<summary>flutter doctor -v</summary> 

```bash
[✓] Flutter (Channel master, 2.1.0-11.0.pre.165, on macOS 11.2.2 20D80 darwin-x64, locale en-GB)
    • Flutter version 2.1.0-11.0.pre.165 at /Users/tahatesser/Code/flutter_master
    • Framework revision 698d438960 (8 hours ago), 2021-03-09 07:53:21 +0100
    • Engine revision 2441c476a6
    • Dart version 2.13.0 (build 2.13.0-116.0.dev)

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Volumes/Extreme/SDK
    • Platform android-30, build-tools 30.0.3
    • ANDROID_HOME = /Volumes/Extreme/SDK
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Volumes/Extreme/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] VS Code (version 1.54.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.20.0

[✓] Connected device (3 available)
    • Taha’s iPhone (mobile) • 00008020-001059882212002E • ios            • iOS 14.4
    • macOS (desktop)        • macos                     • darwin-x64     • macOS 11.2.2 20D80 darwin-x64
    • Chrome (web)           • chrome                    • web-javascript • Google Chrome 89.0.4389.82

• No issues found!
```

</details>

Thanks for filing the issue
